tools/@aws-cdk/prlint/test/lint.test.ts

```diff
@@ -142,8 +156,8 @@ describe('commit message format', () => {
   })
 });
 
-describe.only('ban breaking changes in stable modules', () => {
-  test.only('breaking change in stable module', async () => {
+describe('ban breaking changes in stable modules', () => {
+  test('breaking change in stable module', async () => {
```

Comment:
For some unknown reason, this test was marked as `only` which caused jest to skip all other tests in this file. Removing that here so that the unit tests in this file actually get run.

Reply:
Whoops, that would be because I broke this test and was fixing it for remodel =( sorry